new-lamassu-admin/src/pages/Transactions/DetailsCard.js

```diff
@@ -125,7 +125,9 @@ const DetailsRow = ({ it: tx, timezone }) => {
   const commissionPercentage =
     Number.parseFloat(tx.commissionPercentage, 2) * 100
   const cashInFee = isCashIn ? Number.parseFloat(tx.cashInFee) : 0
-  const fiat = Number.parseFloat(tx.fiat)
+  const fiat = BigNumber(tx.fiat)
+    .minus(cashInFee)
+    .toFixed(2, 1) // ROUND_DOWN
   const crypto = getCryptoAmount(tx)
   const exchangeRate = (fiat / crypto).toFixed(2)
```

Comment:
If I'm not mistaken `.toFixed()` stringifies the number? How does this division even work then?